src/item.cpp

```diff
@@ -14909,7 +14909,7 @@ std::vector<item_comp> item::get_uncraft_components() const
                     continue;
                 }
                 // NOLINTNEXTLINE(clang-analyzer-cplusplus.NewDeleteLeaks)
```

Comment:
Is this compiler instruction still relevant? It was added in #65902 to suppress an apparent false positive, but given that we were copying the items...

Now AFAIK Pass-by-copy into a lambda should create a temporary which goes out of scope and is automatically `delete`d, but I would generally trust clang's opinion? Worth pointing out to you if nothing else

Reply:
Apparently not anymore!